packages/core/utils/src/modules-sdk/migration-scripts/migration-down.ts

```diff
@@ -28,8 +28,8 @@ export function buildRevertMigrationScript({ moduleName, pathToMigrations }) {
   > = {}) {
     logger ??= console as unknown as Logger
 
-    console.log(new Array(TERMINAL_SIZE).join("-"))
```

Comment:
Maybe using console was intentional, but it nevertheless adds a lot of noise so it's better to always use the logger